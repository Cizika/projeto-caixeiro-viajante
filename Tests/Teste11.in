11
1 2 100
1 3 50
1 4 150
1 5 115
1 6 120
1 7 90
1 8 110
1 9 80
1 10 200
1 11 75
2 3 100
2 4 115
2 5 80
2 6 120
2 7 90
2 8 150
2 9 125
2 10 140
2 11 200
3 4 70
3 5 90
3 6 140
3 7 115
3 8 65
3 9 40
3 10 120
3 11 60
4 5 100
4 6 80
4 7 140
4 8 115
4 9 65
4 10 90
4 11 125
5 6 100
5 7 80
5 8 95
5 9 115
5 10 220
5 11 110
6 7 100
6 8 120
6 9 125
6 10 80
6 11 65
7 8 100
7 9 90
7 10 45
7 11 80
8 9 100
8 10 70
8 11 115
9 10 150
9 11 65
10 11 150