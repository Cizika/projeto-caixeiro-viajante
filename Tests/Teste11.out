1
1-2-5-7-10-8-3-9-4-6-11-1
765
